qml/Ping360ControlPanel.qml

```diff
@@ -42,13 +33,83 @@ Item {
                 control.to: 100
                 control.onMoved: ping.range = control.value
             }
-            PingSlider {
-                Layout.fillWidth: true
-                text: "Receiver Gain"
-                value: ping.gain_setting
-                control.from: 0
-                control.to: 2
-                control.onMoved: ping.gain_setting = control.value
+
+            // Uncomment to debug ComboSlider below
+//            PingSlider {
+//                Layout.fillWidth: true
+//                text: "Receiver Gain"
+//                value: ping.gain_setting
+//                control.from: 0
+//                control.to: 2
+//                control.onMoved: ping.gain_setting = control.value
+//            }
+
+            RowLayout {
+                Layout.preferredHeight: 50
+
+                Label {
+                    horizontalAlignment: Text.AlignRight
+                    wrapMode: Text.Wrap
+                    Layout.fillWidth: true
+                    Layout.preferredWidth: 120
+                    text: "Receiver Gain: "
+                }
+
+                ComboSlider {
+                    Layout.fillWidth: true
+                    value: ping.gain_setting
+                    model: ["Low", "Medium", "High"]
+                    onOptionChanged: {
+                        console.log("Value " + value + " at key " + key)
+                        ping.gain_setting = key
+                    }
+                }
+
+                Label {
+                    Layout.fillWidth: true
+                    Layout.preferredWidth: 40
+                    text: ping.gain_setting
+                }
+            }
```

Comment:
Should `PingComboSlider` follow the same structure as `PingSlider` ? Otherwise all this duplicated code is necessary to perform the same job, I believe it should be inside `PingComboSlider` and `PingComboSlider` should inherit `PingSlider` in the end, this will make it more complete and style compatible with `PingSlider`, otherwise we need to do this duplicated code that could not match with the current style.